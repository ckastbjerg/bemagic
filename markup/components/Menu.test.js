/* eslint-env jest */
const Menu = require('./Menu');

test('returns menu markup', () => {
    global.config = require('../../config');

    expect(Menu({
        components: {
            block: {},
            background: {},
        },
    })).toMatchSnapshot();
});
