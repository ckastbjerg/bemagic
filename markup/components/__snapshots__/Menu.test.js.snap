exports[`test returns menu markup 1`] = `
"
    <div class=\"bemagic-menu js-menu\">
        
                <div
                    class=\"bemagic-menu__link js-menu-link is-active\"
                    data-page=\"components\"
                >
                    components
                </div>
            
    </div>
"
`;
